Description: do not participate in the DRBD usage count survey by default
Author: Apollon Oikonomopoulos <apoikos@gmail.com>
Bug-Debian: http://bugs.debian.org/711352
--- a/scripts/global_common.conf
+++ b/scripts/global_common.conf
@@ -3,7 +3,7 @@
 # feature requests visit http://www.linbit.com
 
 global {
-	usage-count yes;
+	usage-count no;
 	# minor-count dialog-refresh disable-ip-verification
 	# cmd-timeout-short 5; cmd-timeout-medium 121; cmd-timeout-long 600;
 }
